radicle-cli/src/commands/inspect.rs

```diff
@@ -122,82 +112,94 @@ pub fn run(options: Options, ctx: impl term::Context) -> anyhow::Result<()> {
         }
     };
 
+    if options.target == Target::Id {
+        term::info!("{}", term::format::highlight(id.urn()));
+        return Ok(());
+    }
+
+    let profile = ctx.profile()?;
+    let storage = &profile.storage;
+    let signer = term::signer(&profile)?;
     let project = storage
         .get(signer.public_key(), id)?
         .context("No project with such `id` exists")?;
 
-    if options.refs {
-        let path = profile
-            .home
-            .storage()
-            .join(id.urn())
-            .join("refs")
-            .join("namespaces");
-
-        Command::new("tree")
-            .current_dir(path)
-            .args(["--noreport", "--prune"])
-            .stdout(Stdio::inherit())
-            .stderr(Stdio::inherit())
-            .spawn()?
-            .wait()?;
-    } else if options.payload {
-        println!(
-            "{}",
-            colorizer().colorize_json_str(&serde_json::to_string_pretty(&project.payload)?)?
-        );
-    } else if options.history {
-        let repo = storage.repository(id)?;
-        let head = Doc::<Untrusted>::head(signer.public_key(), &repo)?;
-        let history = repo.revwalk(head)?;
-
-        for oid in history {
-            let oid = oid?.into();
-            let tip = repo.commit(oid)?;
-            let blob = Doc::<Unverified>::blob_at(oid, &repo)?;
-            let content: serde_json::Value = serde_json::from_slice(blob.content())?;
-            let timezone = if tip.time().sign() == '+' {
-                #[allow(deprecated)]
-                FixedOffset::east(tip.time().offset_minutes() * 60)
-            } else {
-                #[allow(deprecated)]
-                FixedOffset::west(tip.time().offset_minutes() * 60)
-            };
-            let time = DateTime::<Utc>::from(
-                std::time::UNIX_EPOCH + std::time::Duration::from_secs(tip.time().seconds() as u64),
-            )
-            .with_timezone(&timezone)
-            .to_rfc2822();
-
+    match options.target {
+        Target::Refs => {
+            let path = storage.path_of(&id).join("refs").join("namespaces");
+
+            Command::new("tree")
+                .current_dir(path)
+                .args(["--noreport", "--prune"])
+                .stdout(Stdio::inherit())
+                .stderr(Stdio::inherit())
+                .spawn()?
+                .wait()?;
+        }
+        Target::Payload => {
             println!(
-                "{} {}",
-                term::format::yellow("commit"),
-                term::format::yellow(oid),
+                "{}",
+                colorizer().colorize_json_str(&serde_json::to_string_pretty(&project.payload)?)?
```

Comment:
Should we add `term::json`?

Reply:
Yeah probably :), `term::format::json` I guess.